source/game_sa/Radar.cpp

```diff
@@ -1096,21 +1096,20 @@ void CRadar::DrawRadarSprite(uint16 spriteId, float x, float y, uint8 alpha)
         LimitToMap(&x, &y);
     }
 
-    float width = SCREEN_WIDTH_UNIT * 8.0f;
-    float height = SCREEN_HEIGHT_UNIT * 8.0f;
+    uint32 width  = 8 * SCREEN_WIDTH_UNIT;
+    uint32 height = 8 * SCREEN_HEIGHT_UNIT;
```

Comment:
Let's use floor, otherwise the compiler will complain:
```suggestion
    float width = std::floor(SCREEN_WIDTH_UNIT * 8.f);
    float height = std::floor(SCREEN_HEIGHT_UNIT * 8.f);
```